module/VuFind/src/VuFind/Cover/Router.php

```diff
@@ -85,6 +85,32 @@ public function __construct($url, CoverLoader $coverLoader)
      */
     public function getUrl(RecordDriver $driver, $size = 'small',
         $resolveDynamic = true, $testLoadImage = false
+    ) {
+        $metadata = $this->getMetadata(
+            $driver, $size, $resolveDynamic, $testLoadImage
+        );
+        if ($metadata && $metadata['url']) {
```

Comment:
Should there be an `isset` around `$metadata['url']`? Maybe it's okay if the $metadata array is always empty if URL is unset, but if that's the case, then I don't see a reason for the second part of the if condition at all.

Reply:
Alternatively, could this whole block of code be reduced to `return $metadata['url'] ?? $metadata;`? Or is that too much of an oversimplification?

Reply:
the `return $metadata['url'] ?? $metadata;` should be OK, I'll change that